test/runtests.jl

```diff
@@ -221,8 +224,10 @@ scalar_diff = ScalarDiffusivity(ν=1e-6, κ=1e-7)
                                     tracers = :b,
                                     closure = closure)
 
-        @info "Testing energy dissipation rate terms with closure" closure
-        test_ke_dissipation_rate_terms(model)
+        if !(closure isa Tuple) || all(isa.(closure, ScalarDiffusivity{ThreeDimensionalFormulation}))
+            @info "Testing energy dissipation rate terms with closure" closure
+            test_ke_dissipation_rate_terms(model)
+        end
```

Comment:
This if clause is here because `ViscousDissipationRate` doesn't work with anisotropic closures yet, but I'm working on that